Sleep Tracker/SleepTracker/View/ReusableViews/ButtonStyle/RoundedRectangleButton.swift

```diff
@@ -0,0 +1,34 @@
+import SwiftUI
+
+/// A type that applies standard interaction behavior and a custom appearance to all buttons within a view hierarchy.
+/// The button will have a rectanglular bright-blue shape with rounded corners.
```

Comment:
Please correct a grammatical error in your documentation
```suggestion
/// The button will have a rectangular bright-blue shape with rounded corners.
```

Reply:
Fixed.